docs/auth/tokens.rst

```diff
@@ -77,7 +77,7 @@ To delete an existing token by its ID via the token management endpoints, issue
 ``DELETE`` request on the token's endpoint, replacing ``:id`` with the
 token ``id`` value::
 
-    curl -X DELETE https://desec.io/api/v1/auth/tokens/:id/ \
+    curl -X DELETE https://desec.io/api/v1/auth/tokens/{id}/ \
```

Comment:
also in line 77

Reply:
maybe check kind of error for all the replacements

Reply:
yeah, I had been looking for occurrences of `/:` and miss all the other instances that were not preceded by a slash. Fixed them all now.